### ISSUE TITLE
[Bug] Captions returned timedtext xml instead of WebVTT

### ISSUE THREAD

**Issue by iBicha** [iv-org/invidious#3455]
**Describe the bug**
Captions are now returning timedtext xml data instead of WebVTT

**Steps to Reproduce**
1. Open https://yt.funami.tech/api/v1/captions/yy16KFzM4XU?label=English

**Logs**
```
<?xml version="1.0" encoding="utf-8" ?><timedtext format="3">
<head>
<wp id="0"/>
<wp id="1" ap="1" ah="50" av="0" rc="3" cc="40"/>
</head>
<body>
<p t="60" d="1860" wp="1">I am exactly thirty miles away</p>
<p t="1920" d="3800" wp="1">from the intersection of Beverly Boulevard
and North La Cienega Boulevard</p>
...
</body>
</timedtext>
```
**Screenshots**

**Additional context**
Response header has `content-type: text/vtt; charset=UTF-8`

**Comment by DUOLabs333** [comment]
This should be fixed by PR #3533.

**Comment by xatier** [comment]
@DUOLabs333 it works! Thanks so much for this PR!
